src/table.scss

```diff
@@ -124,6 +124,16 @@ $block: #{$fd-namespace}-table;
     }
   }
 
+  &__text {
+    word-break: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
you could use the mixin here

Reply:
@itmilos why did you resolve this comment?

Reply:
One of the Definitions of Done is `Mixins are used for repeatable code (fd-rtl, fd-ellipsis, fd-flex, fd-selected, fd-focus, ect.)` and there's a mixin `fd-ellipsis` that has these 3 rules